src/commands/ReminderCommand.ts

```diff
@@ -23,9 +27,16 @@ class ReminderCommand extends Command {
 	async elapse(client: Client, id: number): Promise<void> {
 		const reminder = reminderDb.get(id);
 		if (!reminder) return;
-		const ch = (await client.channels.fetch(reminder.channel)) as TextBasedChannel;
-		await ch.send({
-			content: `${reminder.callAll ? "<@everyone>" : `<@${reminder.user}>`} Time's up! ${reminder.message}`,
+		const channel = (await client.channels.fetch(reminder.channelID)) as TextBasedChannel;
+		await channel?.send({
+			content: reminder.pings.join(" "),
+			embeds: [
+				addDefaultEmbedFooter(
+					new EmbedBuilder()
+						.setTitle(`${reminder.pings.join(" ")} Time is up!`)
```

Comment:
no point in having pings inside the embed when already outside

Reply:
Yeah I know I just thought they look good there